docs/fundamentals/dashboard/standalone.md

```diff
@@ -19,8 +19,7 @@ The [.NET Aspire dashboard](overview.md) provides a great UI for viewing telemet
 The dashboard is started using the Docker command line.
 
 ```bash
-docker run --rm -it -p 18888:18888 -p 4317:18889 -d --name aspire-dashboard \
-    mcr.microsoft.com/dotnet/aspire-dashboard:8.0.0
+docker run --rm -it -p 18888:18888 -p 4317:18889 -d --name aspire-dashboard mcr.microsoft.com/dotnet/aspire-dashboard:8.0.0
```

Comment:
We strive pretty hard to avoid horizontal scrolling and this change would cause that...We should instead use the compatible command newline delimiters for PowerShell, as this works with Bash.

The syntax for PowerShell is the back tick instead of the backslash.

```powershell
docker run --rm -it `
    -p 18888:18888 `
    -p 4317:18889 -d `
    --name aspire-dashboard `
    mcr.microsoft.com/dotnet/aspire-dashboard:8.0.0
```

Whereas for bash, we could update it to be:

```bash
docker run --rm -it \
    -p 18888:18888 \
    -p 4317:18889 -d \
    --name aspire-dashboard \
    mcr.microsoft.com/dotnet/aspire-dashboard:8.0.0
```

Reply:
yeah, maybe we can put it in tabs?

Reply:
Yes, sir... that's what I'm thinking, do you need help with that or do you know that syntax?